docs/sources/operators-guide/configure/setting-helm-ha-deduplication-consul/index.md

```diff
@@ -0,0 +1,125 @@
+---
+aliases:
+  - /docs/mimir/latest/operators-guide/configuring/setting-helm-ha-deduplication-consul/
+description: Learn how to configure Grafana Mimir Helm Chart to handle HA Prometheus server deduplication with Consul.
+menuTitle: Configuring high-availability deduplication with Consul
+title: Configuring Grafana Mimir Helm Chart for high-availability deduplication with Consul
+weight: 70
+---
+
+# Configuring Grafana Mimir Helm Chart for high-availability deduplication with Consul
```

Comment:
Can we use the name of the chart - mimir-distributed? Maybe a tech writer can also pitch in, but using the name seems more concrete and searchable

Reply:
I will take a look at this soon; on UTC-4 atm and might not get to this until next Monday.